src/org/labkey/test/components/ui/grids/SaveGridViewDialog.java

```diff
@@ -0,0 +1,120 @@
+package org.labkey.test.components.ui.grids;
+
+import org.labkey.test.BootstrapLocators;
+import org.labkey.test.Locator;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.components.html.Input;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Dialog used to save a grid view in an app.
+ */
+public class SaveGridViewDialog extends ModalDialog
```

Comment:
Reference product components.
```suggestion
/**
 * Dialog used to save a grid view in an app. Wraps SaveViewModal.tsx in UI Components.
 */
public class SaveViewDialog extends ModalDialog
```

Reply:
Changed.